web/app/Style.js

```diff
@@ -78,5 +78,14 @@ Ext.define('Traccar.Style', {
     coordinatePrecision: 6,
     numberPrecision: 2,
 
-    reportTagfieldWidth: 375
+    reportTagfieldWidth: 375,
+
+    chartPadding: {
```

Comment:
Unfortunately not. It accepts string http://docs.sencha.com/extjs/6.2.0/classic/Ext.chart.CartesianChart.html#cfg-insetPadding
But takes only first number for all dimensions (bug or feature)